packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1619,3 +1625,271 @@ Multiple IdP instances can be defined in a single Policy Action. This allows use
+
+The following conditions may be applied to the Rules associated with App Sign On Policy:
+
+* [People Condition](#people-condition-object)
+
+* [Network Condition](#network-condition-object)
+
+* [Device Condition](#device-condition-object)
+
+* [Platform Condition](#platform-condition-object)
+
+* [Expression Language Condition](#el-condition-object)
+
+* [Office365Client Condition](#office365Client-condition-object)
+
+* [Risk Condition](#riskscore-condition-object)
+
+* [User Type Condition](#usertype-condition-object)
+
+#### App Sign On Action default example
+
+```json
+  "actions": {
+        "appSignOn": {
+            "access": "DENY",
+            "verificationMethod": {
+                "factorMode": "1FA",
+                "type": "ASSURANCE",
+                "reauthenticateIn": "PT43800H"
+            },
+            "oktaSignOnSettings": null
+        }
+    }
+```
+
+#### App Sign On Action object
+
+| Property                | Description                                                                                                                                                               | Data Type                                       | Required                      | Default |
+| ---                     | ---                                                                                                                                                                       | ---                                             | ---                           | ---     |
+| `access`                  | `ALLOW` or `DENY`                                                                                                                                                         | `ALLOW` or `DENY`                               | Yes                           | N/A     |
+| `verificationMethod`      | A Verification Method describes the means by which the user must be verified. The only supported type is ASSURANCE.                                                       | Object                                          | Yes                           | [Default](#app-sign-on-action-default-example)        |
+| `oktaSignOnSettings`      | TODO                                                                                                                                                                       | TODO                                            | Yes                           | null    |
+
+
+### Verification Method Object
+
+A Verification Method describes the means by which the user must be verified. The only supported type is `ASSURANCE`.
+
+Assurance is the degree of confidence that the end-user logging into an application or service is the same end-user who previously enrolled or logged in to the application or service.
+
+Authenticators can be broadly classified into factors. A factor represents the mechanism by which an end-user owns or controls the authenticator. The three classifications are:
```

Comment:
```suggestion
Authenticators can be broadly classified into factors. A factor represents the mechanism by which an end-user owns or controls the authenticator. The three classifications are:
```

You have to classify into at least two things. Maybe the better question is: what is the difference between a factor and an authenticator?

Reply:
This part is copied from https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-site/docs/reference/api/oie-policy-types/index.md, factors and authenticators are same, authenticators are introduced recently to replace old factors.

Reply:
The term factor is still used in OIE but now `a factor` is the classification/type of an authenticator. Whereas an authenticator is what a user actually uses to verify their identity.

Reply:
cc @thanh-hanguyen-okta @royluo-okta Is the dev doc fine as is or should it explain more about what an authenticator is vs what a factor is? Perhaps we have it defined in some other doc that we could link here?

Reply:
I think if we did want to include more of an explanation, it should go in the Authenticators Admin section page.